doc/classes/GraphNode.xml

```diff
@@ -301,6 +317,8 @@
 		<theme_item name="title_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
 			Color of the title text.
 		</theme_item>
+		<theme_item name="close_h_offset" data_type="constant" type="int" default="22">
```

Comment:
Would be worth documenting.

`close_offset` should maybe be renamed to `close_v_offset` for consistency.

Reply:
Fixed.